support/modules/rootfs-image

```diff
@@ -5,10 +5,27 @@ set -ue
 STATE="$1"
 FILES="$2"
 
+if [ -n "$(which jq >/dev/null 2>&1)" ]; then
+    JQ_AVAILABLE=1
+else
+    JQ_AVAILABLE=0
+fi
+
 CONF_FILE="/var/lib/mender/mender.conf"
 
-MENDER_ROOTFS_PART_A="$(jq -r .RootfsPartA < "$CONF_FILE")"
-MENDER_ROOTFS_PART_B="$(jq -r .RootfsPartB < "$CONF_FILE")"
+if [ "$JQ_AVAILABLE" = 1 ]; then
```

Comment:
No need to quote

Reply:
On the same note, we should also use `{}` for variables then

Reply:
AFAIK, it has no practical effect, it's more of a style issue. I don't mind it, but I'm not going to change the whole file now.